scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -383,7 +383,7 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
     {% set duplicated_variants = [] %}
     {% if variants.classified_detailed %}
       {% for variant in variants.classified_detailed|sort(attribute='variant_rank') %}
-        {% if variant['_id'] not in printed_vars %}
+        {% if variant['_id'] not in printed_vars and variant not in variants.dismissed_detailed %}
```

Comment:
💯 
Not sure it does that by hash really, but it should never be more than a few tens of variants anyway. 🤷